.github/workflows/release.yml

```diff
@@ -195,7 +195,7 @@ jobs:
       - name: Smoke test
         shell: bash
         env:
-          RPC_URL: ${{ secrets.CHEATNET_RPC_URL }}
+          NODE_URL: ${{ secrets.NODE_URL }}
```

Comment:
nitpick: we could just pass the whole url to the script (atm we construct it basen on node ip in smoke_test.sh script), it may potential updates to CIs just a bit easier.